marin/speedrun/create_leaderboard.py

```diff
@@ -10,17 +10,34 @@
 import fsspec
 
 
+@dataclass(frozen=True)
+class SpeedrunAuthor:
+    """Author information displayed in the leaderboard."""
+    name: str
+    affiliation: str
+    url: str | None = None
+
+
 @dataclass(frozen=True)
 class LeaderboardEntry:
+    # Run identification
     run_name: str
-    model_size: int
-    total_training_time: float
-    total_training_flops: float
-    submitted_by: str
     results_filepath: str
-    wandb_link: str | None = None
+
+    # Model/hardware specs
+    model_size: int
+    training_hardware_flops: float
+
+    # Training metrics and FLOPs
+    training_time_in_minutes: float
```

Comment:
Can we make this `training_time` and make it seconds, since I think it's easier if everything is just in seconds (e.g., flops).